packer/http/ks-student.cfg

```diff
@@ -1,8 +1,8 @@
 # kickstart host is student
 # vm hostname is student.puppetlabs.vm
 # product: centos
-# version: 7
-# arch: x86-64
+# version: 6
+# arch: i386
```

Comment:
32bit student VM.  I had renamed it "legacy" but we decided to not go that route.  We'll keep this around until we're ready to retire the 32bit student VM.